jwst/tests_nightly/general/pipelines/test_nrc_tso3_1.py

```diff
@@ -0,0 +1,195 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_tso3 import Tso3Pipeline
+# DISABLED
+# import pandokia.helpers.filecomp as filecomp
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_tso3_pipeline1(_bigdata):
+    """Regression test of calwebb_tso3 pipeline on NIRCam simulated data.
+
+    Default imaging mode outlier_detection will be tested here.
+    """
+    testname = "test_tso3_pipeline1"
+    # Define where this test data resides in testing tree
+    subdir = 'pipelines/nircam_caltso3/'  # Can NOT start with path separator
+
+    # You need to have a tda dict for:
+    #  - recording information to make FlagOK work
+    #  - recording parameters to the task as attributes
+    global tda
```

Comment:
Maybe future work, but the tests here need some clean up to remove unnecessary `pandokia` artifacts.